Formula/rosa-cli.rb

```diff
@@ -0,0 +1,29 @@
+class RosaCli < Formula
+  desc "RedHat OpenShift Service on AWS (ROSA) command-line interface"
+  homepage "https://www.openshift.com/products/amazon-openshift"
+  url "https://github.com/openshift/rosa/archive/refs/tags/v1.0.1.tar.gz"
+  sha256 "589c105402e9b45034969d86a04a904482eb04d03590e5acaf103011478feee6"
+  license "Apache-2.0"
+
+  head "https://github.com/openshift/rosa.git"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+    regex(%r{href=.*?/tag/v?(\d+\.\d+\.\d+)["' >]}i)
+  end
+
+  depends_on "coreutils" => :build
+  depends_on "go" => :build
+  depends_on "awscli"
+
+  def install
+    system "go", "build", *std_go_args, "./cmd/rosa"
```

Comment:
```suggestion
    system "go", "build", *std_go_args, "-o", bin/"rosa", "./cmd/rosa"
```
Oops. Sorry, wasn't paying attention.